tools/mipgen/src/main.cpp

```diff
@@ -71,13 +71,13 @@ If the output format is a container format like KTX, then
    --license, -L
        print copyright and license information
    --linear, -l
-       assume that image pixels are already linearized
+       specifies that the source image transforms linearly to floats
```

Comment:
What do you mean by this? That the data is linear or that it will go through an transfer function?

Reply:
Heh, I thought I was making it _more_ clear but you're right it has "transform" in there which is confusing.  I meant transform in the sense of bytes-to-floats.  How about we say simply: "specifies that the source image is linear".

Reply:
you can say "converts to float" instead of "transform"?